### DISCUSSION TITLE
Sort_Title option?

### DISCUSSION THREAD

**Discussion by ch1ma3ra** [advplyr/audiobookshelf#1010] [Ideas]
A number of media management tools allow you to set "sort_$whatever" metadata field in addition to the basic ones (so title and sort_title, artist and sort_artist etc) - allowing you greater control of how stuff is sorted to put it in the order of your choice without having to mangle the "visible" names - for example I would prefer in the main library view to have "The Hobbit" followed by all of the Lord of the Rings books in the correct order but currently they are all scattered across my library according to their actual book titles.

This would also be useful for multiple series of books that are interrelated but shouldn't be combined into a single series, for example Star Wars novels.

**Comment by advplyr** [comment]
I created an issue for this #1024. I think we should do this but I think we need to reconsider the UI/UX in the details tab when editing books. It is getting to be a lot of inputs there.
